src/command/cmd_funcs.c

```diff
@@ -7579,7 +7589,60 @@ cmd_pgp(ProfWin* window, const char* const command, gchar** args)
     }
 
     if (g_strcmp0(args[0], "sendfile") == 0) {
-        _cmd_set_boolean_preference(args[1], command, "Sending unencrypted files using /sendfile while otherwise using PGP", PREF_PGP_SENDFILE);
+        _cmd_set_boolean_preference(args[1], "Sending unencrypted files using /sendfile while otherwise using PGP", PREF_PGP_SENDFILE);
+        return TRUE;
+    }
+
+    if (g_strcmp0(args[0], "sendpub") == 0) {
+        jabber_conn_status_t conn_status = connection_get_status();
+        if (conn_status != JABBER_CONNECTED) {
+            cons_show("You must be connected to share your PGP public key.");
+            return TRUE;
+        }
+
+        if (window->type != WIN_CHAT && args[1] == NULL) {
+            cons_show("You must set recipient in an argument or use this command in a regular chat window to share your PGP key.");
+            return TRUE;
+        }
+
+        ProfAccount* account = accounts_get_account(session_get_account_name());
+
+        if (account->pgp_keyid == NULL) {
+            cons_show_error("Please, set the PGP key first using /account set %s pgpkeyid <pgpkeyid>. To list pgp keys, use /pgp keys.", account->name);
+            account_free(account);
+            return TRUE;
+        }
+        cons_show("account->pgp_keyid = %s", account->pgp_keyid);
```

Comment:
That's more like a debug statement, so this can go. Or should it stay? Then please give it a more descriptive message that's understandable for a regular user.

Reply:
Debug statement, I guess.